src/pages/admin/reports/components/funds-expenditures-section/components/common/delete-funds-expenditures-category-modal/DeleteFundsExpendituresCategoryModal.module.scss

```
@@ -0,0 +1,146 @@
+        flex-wrap: nowrap;
+        justify-content: center;
+    }
+
+    :global(.modal-footer button) {
+        width: 100%;
+    }
+}
+
+.title {
+    @include type.h4-bold;
+    line-height: 36px;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)